flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkTableSource.java

```diff
@@ -542,14 +542,24 @@ public void testFilterPushDownLike() {
     Assert.assertEquals("Should create only one scan", 1, scanEventCount);
     Assert.assertEquals(
         "Should contain the push down filter", expectedFilter, lastScanEvent.filter().toString());
+
+    sqlLike = "SELECT * FROM  " + TABLE_NAME + "  WHERE data LIKE '%%' ";
```

Comment:
Currently Flink 1.17 can push this down.

Reply:
we should move this to a separate test method

Reply:
`testFilterPushDownLike()`  is used to test the `like filter` that can be pushed down, and we seem to have migrated it here from `testFilterNotPushDownLike()` just fine.